pages/common/tr.md

```diff
@@ -1,31 +1,27 @@
 # tr
 
-> Translate characters: run replacements based on single characters and character sets.
+> Translate characters: the tr utility will translate (i.e., substitute or delete) charaters from an input stream and output those results.
 
-- Replace all occurrences of a character in a file, and print the result:
+- Replace all occurrences of a character in a file, and print the result on the screen (file remains unchanged):
 
 `tr {{find_character}} {{replace_character}} < {{filename}}`
 
 - Replace all occurrences of a character from another command's output:
 
 `echo {{text}} | tr {{find_character}} {{replace_character}}`
 
-- Map each character of the first set to the corresponding character of the second set:
-
-`tr '{{abcd}}' '{{jkmn}}' < {{filename}}`
-
-- Delete all occurrences of the specified set of characters from the input:
+- Delete all occurrences of the specified set of characters from the input file and output to the screen:
 
 `tr -d '{{input_characters}}' < {{filename}}`
 
-- Compress a series of identical characters to a single character:
+- Compress a series of identical characters in a file to a single character and output to the screen:
```

Comment:
I don't believe "and output to the screen" is needed after the first time. It's pretty much understood.

Reply:
I agree. However, I was afraid the examples would lead new users of the cmd to believe that it updated the files the examples were acting on. It simply outputs the updates to the screen. I'll update them all.